src/zmq/zmqnotificationinterface.cpp

```diff
@@ -29,6 +29,14 @@ CZMQNotificationInterface::~CZMQNotificationInterface()
     }
 }
 
+std::list<const CZMQAbstractNotifier*> CZMQNotificationInterface::GetActiveNotifiers() const
+{
+    std::list<const CZMQAbstractNotifier*> result;
```

Comment:
Could just `return notifiers`?

Reply:
No because `notifiers` is a list of *non-`const`* `CZMQAbstractNotifier`, and I'd rather not return the mutable objects.  That's the reason why I originally had this "add to JSON" method - but I think that the additional copy done here is probably not relevant in practice.